open_feature/open_feature_client.py

```diff
@@ -14,8 +25,8 @@ def __init__(
         self,
         name: str,
         version: str,
-        context=None,
-        hooks: list = None,
+        context: EvaluationContext = EvaluationContext(),
```

Comment:
I wonder if the `evaluation_context` module should export a constant like `EMPTY_EVALUATION_CONTEXT` to handle this? Although, actually I think that might not work. My concern is that I have a feeling python will (loosely) complain about this as we're setting a mutable argument as the default here. 

We may need to do: 

```python
def __init__(..., context: EvaluationContext = None, ...):
    ...
    self.evaluation_context = context or EvaluationContext()
```

Reply:
The problem with this is all the merges fail if the evaluation context is not set to something

Reply:
So the issue (I think) with what you had previously is that the default argument is evaluated when the module is loaded into memory, so if multiple instances of the OpenFeatureClient are instantiated without a `context` argument, they'll actually end up sharing the `EvaluationContext` object that was instantiated when the module was first loaded. Any changes made to the context in one client would also be seen in the other. 

That's my understanding at least. 

Looks like you've changed it to what I suggested anyway so I'll go ahead and resolve this conversation.